services/IoT/applications/libraries/java.md

```diff
@@ -376,14 +376,14 @@ For the complete code sample, go to the following application example:
 
 [HttpApplicationDeviceEventPublish](https://github.com/ibm-messaging/iot-java/blob/master/samples/iotfdeviceclient/src/com/ibm/iotf/sample/client/application/HttpApplicationDeviceEventPublish.java)
 
-Based on the settings in the properties file, the publishEventOverHTTP() method publishes the event in either Quickstart or in Registered flow. When  `quickstart` is the Organization ID that is specified in the properties file, the publishEventOverHTTP() method publishes the event to the  {{site.data.keyword.iot_short_notm}} Quickstart service in plain HTTP format. When a valid registered organization is specified in the properties file, the event is always published through HTTPS, so that all of the communication is secure.
+Based on the settings in the properties file, the publishEventOverHTTP() method publishes the event in either Quickstart or in Registered flow. When  `quickstart` is the Organization ID that is specified in the properties file, the publishEventOverHTTP() method publishes the event to the {{site.data.keyword.iot_short_notm}} Quickstart service in plain HTTP format. When a valid registered organization is specified in the properties file, the event is always published by using HTTPS, so that all of the communication is secure.
 
-The event in HTTP(s) is published at most once Quality of Service, so the application needs to implement the retry logic when there is an error.
+The event in HTTP is published at most once Quality of Service, so the application needs to implement the retry logic when there is an error.
```

Comment:
"The event in HTTP is published at most once Quality of Service,"
this phrase does not read well. "At most once quality of service" does not make sense, and I'm having a hard time guessing what it might mean. Clarify.